### DISCUSSION TITLE
Customize email template for password reset notification

### DISCUSSION THREAD

**Discussion by mhanifmuhsin** [inertiajs/inertia#1007] [Help (React)] [answered]
I have a case to change the default email template from Laravel, are there friends who have custom email templates in Inertia js and can show the references ?
- the picture below is the case in question
<img width="600" alt="Screen Shot 2021-12-13 at 10 00 26" src="https://user-images.githubusercontent.com/68141840/145745684-4c40945e-69ab-4f26-8283-8c25c53fac8e.png">

**Comment by ajnsn** [comment, marked as answer]
This is more of a Laravel specific question and not something specific to Inertia.
You can find more about reset email customization [here](https://laravel.com/docs/8.x/passwords#reset-email-customization)

Alternatively you can ask e.g. at [Laracasts Forums](https://laracasts.com/discuss)

**Reply by mhanifmuhsin** [reply]
Thanks for the information
